tests/test_manufactured.py

```diff
@@ -30,7 +36,7 @@ def runTest(self):
         def boundary_flux(v, dv, w):
             return v * (w.x[0] == 1.)
 
-        n = m.p.shape[-1]
+        _ = m.p.shape[-1]
```

Comment:
These lines look like they could be leftovers from the past and be removed.

Reply:
Yes, I don't see any point in those. I removed the lines.